src/scenes/level.js

```diff
@@ -82,16 +82,15 @@ export class LevelScene extends Phaser.Scene {
     }
 
     init(characterData) {
-        // Always receive character class
-        this.characterData.characterClass = characterData.characterClass;
-
         // Receive cookies if they exist
         if (characterData.hasCookies) {
             this.characterData = characterData;
         }
         // Otherwise, initialize character based on starting class
         else {
-            switch (this.characterData.characterClass) {
+            // Reset data
+            this.characterData = JSON.parse(JSON.stringify(saveData));
```

Comment:
It is needed unless you can think of a better way to do it. Resetting it to null will cause errors when you try to reference the data in the second level of the object. This needs to be here for the reset cookies button to work otherwise it will maintain the previous data

Reply:
I'm saying that aren't we already initializing to this on line 49?

Reply:
OH for the reset cookies button. I see i see. Can we initialize to null on line 49?

Reply:
Yeah that'll work